.github/CONTRIBUTING.md

```diff
@@ -648,13 +654,13 @@ responsible for properly tagging new pull requests and issues, moderating commen
 pull requests/issues, and merging/closing pull requests.
 
 ### Maintainer List
-* [Fox P McCloud](https://github.com/Fox-McCloud)
-* [Crazy Lemon](https://github.com/Crazylemon64)
-* [Ansari](https://github.com/variableundefined)
 * [AffectedArc07](https://github.com/AffectedArc07)
+* [Ansari](https://github.com/variableundefined)
+* [Crazy Lemon](https://github.com/Crazylemon64)
+* [Fox P McCloud](https://github.com/Fox-McCloud)
 
 ### Maintainer instructions
-* Do not `self-merge`; this refers to the practice of opening a pull request, then
+* Do not "self-merge"; this refers to the practice of opening a pull request, then
   merging it yourself. A different maintainer must review and merge your pull request, no
   matter how trivial. This is to ensure quality.
   * A subset of this instruction: Do not push directly to the repository, always make a
```

Comment:
For clarity, the only changes made to this section were putting the 'Maintainer list' into alphabetical order, and replacing the backticks with quotation marks around "self-merge".